pkg/pointerdb/pointerdb.go

```diff
@@ -116,6 +137,10 @@ func (s *Server) Get(ctx context.Context, req *pb.GetRequest) (resp *pb.GetRespo
 		return nil, err
 	}
 
+	if err = s.appendSignature(ctx); err != nil {
```

Comment:
Is this something we'll have to manually add to every call or is there a way we can just have this automatically run with every response the server sends out

Reply:
Current version assumes that not all responses will produce signature. For example List request will end at uplink level and won't go to piecestore. In this PR only Get appends a signature because it only request that is always triggered before other uplink requests to piecestore.